packages/dashboard-frontend/src/components/WorkspaceProgress/CreatingSteps/Initialize/__tests__/index.spec.tsx

```diff
@@ -41,6 +41,20 @@ jest.mock('@/services/helpers/location', () => ({
   buildUserPreferencesLocation: (tab: UserPreferencesTab) => 'user-preferences?tab=' + tab,
 }));
 
+// const mockGet = jest.fn().mockReturnValue('all');
+// jest.mock('@/services/session-storage', () => {
+//   return {
+//     __esModule: true,
+//     default: {
+//       get: (...args: unknown[]) => mockGet(...args),
+//     },
+//     // enum
+//     SessionStorageKey: {
+//       TRUSTED_SOURCES: 'trusted-sources', // 'all' or 'repo1,repo2,...'
+//     },
+//   };
+// });
+
```

Comment:
should we delete or fix it ?

Reply:
this are leftovers from the prev implementation, I'll delete them in the next commit.

Reply:
done